src/PrestaShopBundle/Bridge/Helper/Listing/HelperBridge/HelperListBridge.php

```diff
@@ -512,4 +503,32 @@ private function shouldLimitSqlResults($limit): bool
     {
         return $limit !== false;
     }
+
+    /**
+     * @param HelperListConfiguration $helperListConfiguration
+     * @param HelperList $helper
+     */
+    private function setHelperDisplay(
+        HelperListConfiguration $helperListConfiguration,
+        HelperList $helper
+    ): void {
+        $breadcrumbs = $this->breadcrumbsAndTitleConfigurator->getBreadcrumbs($helperListConfiguration->getTabId());
+
+        $helper->title = $breadcrumbs['tab']['name'];
```

Comment:
```suggestion
        $helper->title = $breadcrumbs['tab']['name'] ?? '';
```

I guess we need to do the same as here https://github.com/PrestaShop/PrestaShop/blob/b75c68bea2082f2033518cd2c1bfdff96edd99af/src/PrestaShopBundle/Bridge/Smarty/BreadcrumbsAndTitleConfigurator.php#L79